internal/saucecloud/cloud.go

```diff
@@ -618,10 +618,10 @@ var (
 	otherAppsUpload    uploadType = "other applications"
 )
 
-func (r *CloudRunner) uploadProjects(filename []string, pType uploadType, dryRun bool) ([]string, error) {
+func (r *CloudRunner) uploadProjects(filename, descriptions []string, pType uploadType, dryRun bool) ([]string, error) {
 	var IDs []string
-	for _, f := range filename {
-		ID, err := r.uploadProject(f, pType, dryRun)
+	for i, f := range filename {
+		ID, err := r.uploadProject(f, descriptions[i], pType, dryRun)
```

Comment:
This looks like something that can blow up easily. ⚠️ 
Descriptions are optional. So if I have `otherApps` defined, but provide no descriptions for them, this line will lead to a slice out of bounds error.

Reply:
Added some checks in validation stage and assignment